### PR TITLE
Add groovydoc for missing methods/classes, publish docs from Jenkins

### PR DESCRIPTION
**Pull request by nre-ableton**
ptal @AbletonDevTools/gotham-city, thanks!

### CODE REVIEW

File: src/com/ableton/PythonBuilder.groovy [hunk 1]
```diff
@@ -36,6 +44,11 @@ class PythonBuilder implements Serializable {
    */
   String revision = ''
 
+  /**
+   * Downloads, compiles, and installs Python using values in the fields of this class.
```
Comment (ala-ableton) [inline src/com/ableton/PythonBuilder.groovy]: `Downloads, compiles, and installs` -> `Download, compile, and install`

File: src/com/ableton/VirtualEnv.groovy [hunk 2]
```diff
@@ -6,10 +6,27 @@ package com.ableton
...
   @SuppressWarnings('FieldTypeRequired')
   def script
+  /**
+   * Destination directory for the virtualenv. This value is set during construction of
+   * the object, and is under the job's temporary working directory.
+   */
   String destDir
 
+  /**
+   * Constructs a new instance of this class. This method <strong>does not</strong>
```
Comment (ala-ableton) [inline src/com/ableton/VirtualEnv.groovy]: `Constructs` -> `Construct`

File: src/com/ableton/VirtualEnvList.groovy [hunk 3]
```diff
@@ -1,15 +1,36 @@
...
   @SuppressWarnings('FieldTypeRequired')
   def script
+  /**
+   * List of {@link com.ableton.VirtualEnv} objects.
+   * @see #add(String)
+   */
   List<VirtualEnv> venvs = []
 
+  /**
+   * Creates a new {@link com.ableton.VirtualEnv} and adds it to the list.
```
Comment (ala-ableton) [inline src/com/ableton/VirtualEnvList.groovy]: `Creates` -> `Create`, and `adds` -> `add`

File: src/com/ableton/VirtualEnvList.groovy [hunk 4]
```diff
@@ -1,15 +1,36 @@
...
+   * @param python Python version or absolute path to Python executable.
+   * @see com.ableton.VirtualEnv#create(Object, String)
+   */
   void add(String python) {
     venvs.add(VirtualEnv.create(script, python))
   }
 
+  /**
+   * Runs the given {@code command} in all VirtualEnv instances.
```
Comment (ala-ableton) [inline src/com/ableton/VirtualEnvList.groovy]: `Runs` -> `Run`

File: vars/virtualenv.groovy [hunk 5]
```diff
@@ -1,6 +1,11 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version to use.
```
Comment (ala-ableton) [inline vars/virtualenv.groovy]: We should maybe rename that variable to `pythonVersion`?

Comment (nre-ableton) [reply vars/virtualenv.groovy]: No, since this value can also be a path. I've fixed the documentation accordingly.

**Comment by ala-ableton** [comment]
LGTM 👍

**Pull request merged**